WAW.API/Chat/Resources/ChatRoomResource.cs

```diff
@@ -15,4 +16,7 @@ public class ChatRoomResource {
 
   [SwaggerSchema("Chat room last update date", Nullable = false)]
   public IList<User> Participants { get; set; } = new List<User>();
+
+  [SwaggerSchema("Chat room last update date", Nullable = false)]
+  public IList<Message> Messages { get; set; } = new List<Message>();
```

Comment:
Messages shouldn't be returned here. They should be available in their own endpoint. Something like this:

```http
GET /api/v1/chatroom/{id}/messages
```